components/slider/Slider.js

```diff
@@ -10,6 +10,13 @@ import events from '../utils/events';
 import InjectProgressBar from '../progress_bar/ProgressBar';
 import InjectInput from '../input/Input';
 
+const KEYS = {
```

Comment:
i know `keyCode` was already being used here, but since you were cleaning it up thought i'd just point out that [keyCode is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) and should probably be cleaned up in the future 😸 and replaced with [e.code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code)

Reply:
Good catch! Thank you!

Reply:
looks like you updated to use the `code` property but forgot to update the values in your `KEYS` map.
Should be
```
const KEYS = {
  ENTER: 'Enter',
  ESC: 'Escape',
  ARROW_UP: 'ArrowUp',
  ARROW_DOWN: 'ArrowDown',
};
```
Just for reference, good site for determining values for `code` - https://w3c.github.io/uievents/tools/key-event-viewer.html

Reply:
Ooops, what a fail 🤦🏻‍♂️